pysatMissions/instruments/missions_sgp4.py

```diff
@@ -56,19 +61,24 @@ def init(self):
         'August 21–24.'))
     logger.info(self.acknowledgements)
 
+    if 'epoch' not in self.kwargs['load'].keys():
+        self.kwargs['load']['epoch'] = self.files.files.index[0]
```

Comment:
```suggestion
        self.kwargs['load']['epoch'] = _test_dates['']['']
```
This works locally for me.  It solves the same issue for pysatMissions (consistent epoch for multiple days), but does not require `files` to be generated.

Reply:
Tagging @rstoneback

Reply:
What if someone changes the supported time range? Won't that cause an issue?

My suggestion is to change it to 
```
self.kwargs['load']['epoch'] = self.files.start_date
```
since that attribute exists. Independent of your use the bug in pysat needs to be fixed :)

Reply:
Checking on the suggestion, marked resolved but unchanged with a thumbs up.